core/src/test/java/io/questdb/test/griffin/SecurityTest.java

```diff
@@ -313,6 +317,45 @@ public void testCircuitBreakerTimeout() throws Exception {
         });
     }
 
+    @Test
```

Comment:
has anyone checked if this test is flapping or not? tests around cancellation are notorious hard to make stable